src/uv_tcp_connect.c

```diff
@@ -57,7 +57,7 @@ static int uvTcpEncodeHandshake(raft_id id, const char *address, uv_buf_t *buf)
                sizeof(uint64_t) + /* Server ID. */
                sizeof(uint64_t) /* Size of the address buffer */;
     buf->len += address_len;
-    buf->base = RaftHeapMalloc(buf->len);
+    buf->base = RaftHeapCalloc(1, buf->len);
```

Comment:
This one seems unnecessary, since `buf->len` is `64 * 3` bytes, and all these bytes are initialized just below by this code:

```C
    bytePut64(&cursor, UV__TCP_HANDSHAKE_PROTOCOL);
    bytePut64(&cursor, id);
    bytePut64(&cursor, address_len);
```